WowPacketParser/SQL/SQLDatabase.cs

```diff
@@ -104,8 +104,8 @@ private static void LoadBroadcastText()
 
                     var broadcastText = new BroadcastTextEntry()
                     {
-                        MaleText = maleText,
-                        FemaleText = femaleText,
+                        Text = maleText,
+                        Text1 = femaleText,
```

Comment:
Why change from "FemaleText" to "Text1" ?

Reply:
Original Blizzard naming

Reply:
maleText and co is fixed in other PR